dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/2168516a-5c9a-4582-90dc-5e3a01e3f607.json

```diff
@@ -9,23 +9,29 @@
     "additionalProperties": false,
     "properties": {
       "host": {
+        "description": "Hostname of the database.",
         "type": "string"
       },
       "port": {
+        "description": "Port of the database.",
         "type": "integer",
         "minimum": 0,
         "maximum": 65536
       },
       "user": {
+        "description": "Username to use to access the database.",
         "type": "string"
       },
       "password": {
+        "description": "Password associated with the username.",
         "type": "string"
       },
       "dbname": {
+        "description": "Name of the database.",
         "type": "string"
       },
       "filter_dbs": {
+        "description": "This should be identical to dbname field.",
```

Comment:
Is this going to be filled in from the UI? ideally not but fine if we need to do it for MVP

Reply:
yeah. i think we should do what you described but definitely out of scope of this PR.